### ISSUE TITLE
Passbase 1.9.5 fails to build in Android

### ISSUE THREAD

**Issue by kevinchou723** [passbase/rn-passbase#3]
mobile platfrom: Android SDK 29
react-native-version: 0.62.2
rn-passbase: 1.9.5

CrashLog
```
error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup. Run CLI with --verbose flag for more details.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/@react-native-community/async-storage/android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/expo-app-loader-provider/android/src/main/java/expo/loaders/provider/AppLoaderProvider.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/expo-constants/android/src/main/java/expo/modules/constants/ConstantsService.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/lottie-react-native/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/lottie-react-native/src/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewManager.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
e: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/@passbase/react-native-passbase/android/src/main/kotlin/com/rnpassbase/passbasemodule/PassbaseModule.kt: (209, 22): Unresolved reference: setTheme
e: /Users/myusername/code/myprojectname/clients/mobile-app/node_modules/@passbase/react-native-passbase/android/src/main/kotlin/com/rnpassbase/passbasemodule/PassbaseModule.kt: (211, 22): Unresolved reference: setTheme
```

This happened after I upgraded rn-passbase from 1.8.1 to 1.9.5.  I followed the instructions in the documentation and changed the minimum verision requirements for the buildscript

Buildscript
```

buildscript {
    ext {
        buildToolsVersion = "29.0.2"
        minSdkVersion = 21
        compileSdkVersion = 28
        targetSdkVersion = 28
        supportLibVersion = "28.0.0"
        kotlinVersion = "1.3.21"
    }
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath('com.android.tools.build:gradle:4.0.0')
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
        classpath 'com.google.gms:google-services:4.2.0'
    }
}

```

Please let me know if additional info is required. Thanks!

**Comment by skorostetskyi** [comment]
Hi @kevinchou723 thanks for the report!
A fix is coming tomorrow.

**Comment by kevinchou723** [comment]
@skorostetskyi any news on this issue?

**Comment by skorostetskyi** [comment]
@kevinchou723 check v1.9.6
https://www.npmjs.com/package/@passbase/react-native-passbase
A fix is deployed. Let me know if you need any help.

**Comment by kevinchou723** [comment]
confirm resolved with 1.9.6 thanks!!
